### PR TITLE
Added missing requirements

### PR DESCRIPTION
**Pull request by agrawalsourav98**

### CODE REVIEW

**Review by isaaccorley — CHANGES_REQUESTED** [review]
These deps should be added to the geo section of setup.cfg not voc.

**Comment by agrawalsourav98** [comment]
> These deps should be added to the geo section of setup.cfg not voc.

The issue is only with the voc section. If you install using `ml-dronbase-data-utils[voc]` you would get import error when using the voc code.

File: setup.cfg [hunk 1]
```diff
@@ -48,6 +48,8 @@ voc =
     MarkupSafe==2.0.1
     numpy
```
[the remark this answers is not in the document]
Comment (agrawalsourav98) [reply setup.cfg]: Added

**Comment by isaaccorley** [comment]
> > These deps should be added to the geo section of setup.cfg not voc.
> 
> 
> 
> The issue is only with the voc section. If you install using `ml-dronebase-data-utils[voc]` you would get import error when using the voc code.

I traced this and it's actually because parse_voc imports bbox so it requires all deps from bbox.

**Review by isaaccorley — APPROVED** [review]

**Pull request merged**